pysatSeasons/avg.py

```diff
@@ -1,372 +1,459 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from __future__ import print_function
-from __future__ import absolute_import
-"""
-Instrument independent seasonal averaging routine. Supports averaging
-1D and 2D data.
+# Copyright (C) 2022, pysat development team
+# Full license can be found in License.md
+# -----------------------------------------------------------------------------
+"""Instrument independent seasonal averaging routine.
+
+Supports bin averaging N-dimensional data over 1D and 2D bin distributions.
+
 """
 
 import numpy as np
 import pandas as pds
-import collections
+import warnings
+
 import pysat
-import pysatSeasons as ssnl
+import pysatSeasons as pyseas
 
 
-def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=False):
-    """Return a 1D median of data_label over a season and label1
+def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=None,
+             return_data=False):
+    """Return a 1D median of nD `data_label` over a season and `label1`.
```

Comment:
```suggestion
    """Cacluate a 1D median of nD `data_label` over a season and `label1`.
```

What do you mean by "over a season and `label1`"?